ui/app/components/auth-config-form/config.js

```diff
@@ -15,39 +15,32 @@ import { waitFor } from '@ember/test-waiters';
  *
  * @example
  * ```js
- * {{auth-config-form/config model.model}}
+ * <AuthConfigForm::Config @model={{this.model.model}} />
  * ```
  *
  * @property model=null {DS.Model} - The corresponding auth model that is being configured.
  *
  */
```

Comment:
Mostly an FYI if we do want to eventually add docfy, the syntax for JSDOC `@example` is to remove the backticks ````js. Essentially the component here should be able to render, for example, a real function if the arg is expecting one. But that's a little more difficult to do with models, anyway, mostly just mentioning as an FYI - no need to make any changes. [docfy PR draft](https://github.com/hashicorp/vault/pull/24092/files) for reference

```js
 *@example
 * <AuthConfigForm::Config @model={{this.model.model}} />
 *
 * @property {DS.Model} model=null - The corresponding auth model that is being configured.
 *
 */